src/OpenCV/ImgProc/MiscImgTransform.hs

```diff
@@ -422,3 +425,21 @@ watershed img markers =
                      , *$(Mat * markersPtr)
                      )
       }|]
+
+{- | Returns 0 if the pixels are not in the range, 255 otherwise. -}
+inRange ::
+     (ToScalar scalar)
+  => Mat ('S [w, h]) channels depth
+  -> scalar -- ^ Lower bound
+  -> scalar -- ^ Upper bound
+  -> CvExcept (Mat ('S [w, h]) ('S 1) ('S Word8))
```

Comment:
Would it be useful to convert the result to `Bool` with [`fromBool`](https://hackage.haskell.org/package/base-4.9.0.0/docs/Foreign-Marshal-Utils.html#v:fromBool)?

A small advantage of the library converting to Bool is that it frees the user from thinking about possible results other than 0 or 255.